tests/foreman/cli/test_repository.py

```diff
@@ -3390,3 +3394,87 @@ def test_positive_syncable_yum_format_repo_import(target_sat, module_org):
     )
     assert repodata['content-counts']['packages'] != 0
     assert repodata['sync']['status'] == 'Success'
+
+
+@pytest.mark.no_containers
+def test_possitive_install_uploaded_rpm_on_host(
+    target_sat, rhel7_contenthost, function_org, function_lce
```

Comment:
Could you look around if we can fit the test logic in an existing test? This is completely non-blocking, especially if we save time using container hosts.

Reply:
After a short research of our tests directory, I did not find any suitable test to put the logic in.